manip_control:
  baudrate: 9600
  device: /dev/ttyUSB0
  speeds:
    arm_rotate: 10
    claw_clamp: 30
    claw_rotate: 20
    claw_lift: 60
    arm_tilt: 60
    arm_lift: 60
